x-pack/plugins/monitoring/public/components/elasticsearch/index/index_react.js

```diff
@@ -0,0 +1,70 @@
+/*
```

Comment:
this is the same as the original (index.js), except i'm no longer passing $scope to `shardAllocation` and using `shardAllocationReact` introduced in the [Node view PR](https://github.com/elastic/kibana/pull/113318)